features/extensions/azure/install.py

```diff
@@ -83,30 +83,46 @@ def install(jazz_stackprefix, jenkins_url, jenkins_user, jenkins_api_token, jazz
         setCredential(jenkins_url, jenkins_user, jenkins_api_token, creds['id'],
                       creds['id'], creds['value'])
 
+
+    # Trigger create-serverless-service api
+    createSSJobUrl = "job/build-pack-api/buildWithParameters?token=jazz-101-job&service_name=create-serverless-service&domain" \
+                   "=jazz&scm_branch=master"
+    startJob(jenkins_url, jenkins_user, jenkins_api_token, createSSJobUrl)
     # Trigger metrics api
     metricJobUrl = "job/build-pack-api/buildWithParameters?token=jazz-101-job&service_name=metrics&domain" \
                    "=jazz&scm_branch=master"
     startJob(jenkins_url, jenkins_user, jenkins_api_token, metricJobUrl)
     # Trigger jazz ui
-    startJob(jenkins_url, jenkins_user, jenkins_api_token, "job/jazz_ui/build?token=jazz-101-job")
+    startJob(jenkins_url, jenkins_user, jenkins_api_token, "job/jazz_ui/buildWithParameters?token=jazz-101-job")
 
 
-def prepare_azure_json(azure_location):
+def prepare_azure_json(azure_location, azure_subscription_id):
     extension_base = "extensions/azure"
     azureConfig = OrderedDict()
     azureConfig['IS_ENABLED'] = True
-    azureConfig['SUBSCRIPTION_ID'] = 'AZ_SUBSCRIPTIONID'
-    azureConfig['CLIENT_ID'] = 'AZ_CLIENTID'
-    azureConfig["PASSWORD"] = 'AZ_PASSWORD'
-    azureConfig["TENANT_ID"] = 'AZ_TENANTID'
-    azureConfig['LOCATION'] = azure_location
-    azureConfig['RESOURCE_GROUPS'] = OrderedDict()
-    azureConfig['RESOURCE_GROUPS']['DEVELOPMENT'] = getTerraformOutputVar("dev_resource_group", extension_base)
-    azureConfig['RESOURCE_GROUPS']['STAGING'] = getTerraformOutputVar("stage_resource_group", extension_base)
-    azureConfig['RESOURCE_GROUPS']['PRODUCTION'] = getTerraformOutputVar("prod_resource_group", extension_base)
-    azureConfig['APIM'] = OrderedDict()
-    azureConfig['APIM']['DEVELOPMENT'] = getTerraformOutputVar("dev_apim", extension_base)
-    azureConfig['APIM']['STAGING'] = getTerraformOutputVar("stage_apim", extension_base)
-    azureConfig['APIM']['PRODUCTION'] = getTerraformOutputVar("prod_apim", extension_base)
-
+    azureAccount = OrderedDict()
+    azureAccount['ACCOUNTID'] = azure_subscription_id
+    azureAccount['ACCOUNTNAME'] = 'azureaccount'
+    azureAccount['SUBSCRIPTION_ID'] = 'AZ_SUBSCRIPTIONID'
+    azureAccount['CLIENT_ID'] = 'AZ_CLIENTID'
+    azureAccount["PASSWORD"] = 'AZ_PASSWORD'
+    azureAccount["TENANT_ID"] = 'AZ_TENANTID'
+    azureRegions = OrderedDict()
+    azureRegions['LOCATION'] = azure_location
+    azureRegions['REGION'] = azure_location.lower().replace(' ','')
```

Comment:
Azure Regions have spaces, isn't removing them going to be an issue?

Reply:
I see the reason for this. Azure has `DisplayName` & `Name` for each region. Ex: `DisplayName` -> `East US 2` & `Name` -> `eastus2`. So, `azureRegions['LOCATION'] ` & `azureRegions['REGION']` are being used to store these values respectively (which makes sense).

@rajeevr2715  Is the user expected to specify `DisplayName` during installation?